streaming/java/streaming-api/src/main/java/org/ray/streaming/python/PythonFunction.java

```diff
@@ -40,21 +40,28 @@
 
   private byte[] function;
   private String moduleName;
-  private String className;
   private String functionName;
   /**
    * FunctionInterface can be used to validate python function,
    * and look up operator class from FunctionInterface.
    */
   private String functionInterface;
 
-  private PythonFunction(byte[] function,
-                         String moduleName,
-                         String className,
-                         String functionName) {
+  /**
+   * Create a python function.
+   *
+   * @param function serialized streaming python function from python driver. If it's null,
+   *     then function is constructed from moduleName and functionName.
+   * @param moduleName streaming python function module name. <code>moduleName<code/> will be null
```

Comment:
```suggestion
   * @param moduleName streaming python function module name. {@code moduleName} will be null
```

Otherwise failed with checkstyle violation.